Cargo.toml

```diff
@@ -6,5 +6,5 @@ edition = "2018"
 license = "Apache-2.0 OR MIT"
 
 [dependencies]
-libc = "0.2.71"
+libc = "0.2.94"
```

Comment:
This should probably just be `0.2` right?

Reply:
We agreed a while back that we'd put full version numbers in manifests, as all of the automated tools do (for example I used `cargo upgrade` for this change).

Reply:
I don't think we agreed to that because I think it's generally silly ;) However, I'm not going to argue.